idb.js

```diff
@@ -1,4 +1,9 @@
+const isBrowser = typeof window !== 'undefined'
```

Comment:
this would not work with `Deno`
I think it's better to check if we have `indexedDB`:
```js
const hasIDB = typeof indexedDB !== 'undefined'
```

Reply:
implemented this change first, it works as a direct replacement for my previous isBrowser varable. :)

i'll move on to testing your second change request although i'm a little worried about that one